.github/CODEOWNERS

```diff
@@ -1,40 +1,54 @@
-# The following volunteers have self-identified as subject matter experts
-# or interested parties over a particular area of the php-src source code.
-# While requesting a review from someone does not obligate that person to
-# review a pull request, these reviewers might have valuable knowledge of
-# the problem area and could aid in deciding whether a pull request is ready
-# for merging.
+# The following volunteers have self-identified as subject matter experts or
+# interested parties in a particular area of the php-src source code. While
+# requesting a review from someone does not obligate that person to review a
+# pull request, these reviewers might have valuable knowledge of the problem
+# area and could aid in deciding whether a pull request is ready for merging.
+#
+# When changing this file, please make sure to commit the changes to the
+# earliest supported PHP branch (PHP-X.Y) and not only to the master branch.
```

Comment:
Maybe mention that this is supposed to be handled like bug fixes.

Reply:
Added some sentence about that.

Reply:
On the second thought I'm not sure if these "bug fixes" mention is more understandable what the point is here :)